book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ]  The `:rentals` should be writable also, kindly make sure to add it to `attr_accessor` so it is accessible (read/write) by the class